src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesigner.cs

```diff
@@ -94,7 +99,7 @@ public override bool RunTask ()
 			CodeTypeDeclaration resources;
 			if (UseManagedResourceGenerator) {
 				var parser = new ManagedResourceParser () { Log = Log, JavaPlatformDirectory = javaPlatformDirectory, ResourceFlagFile = ResourceFlagFile };
-				resources = parser.Parse (ResourceDirectory, AdditionalResourceDirectories?.Select (x => x.ItemSpec), IsApplication, resource_fixup);
+				resources = parser.Parse (ResourceDirectory, RTxtFile ?? string.Empty, AdditionalResourceDirectories?.Select (x => x.ItemSpec), IsApplication, resource_fixup);
```

Comment:
Given the "multi-threading" concerns elsewhere (e.g. 51f56aacd9946231672f8c173e265bfe68cb52da), shouldn't this use `Path.GetFullPath(x.ItemSpec)`?

Reply:
Ditto `RTxtFile`, which likewise won't be an absolute path.

Reply:
I think the `GetFullPath()` calls would only be needed if this were an `AsyncTask` and this code was on a background thread.

`GenerateResourceDesigner` looks to be a regular `Task`.

Reply:
I agree this is not an AsyncTask so the node will not be released to another task and then reused. This task will keep hold of the node until the task is complete. So I don't think we need the full paths here.